kr20.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np 
+import json
+
+from flask import Flask, escape, request
+from flask_cors import CORS
+
+app = Flask(__name__)
+CORS(app)
+
+
+@app.route('/')
+def welcome():
+    return {'message': 'Hello'}
+
+
+@app.route('/kr20/<json_array>', methods=['POST', 'GET'])
+def compute_kr20(json_array):
+    req = request.get_json(json_array)
+    return calculate_kr20(req)
+
+
+def calculate_kr20(param):
+    student_list = list(param['students'])
+    numStudents = len(student_list)
+    numQ = len(student_list[0]['itemresponses'])
```

Comment:
You can make it such that any key for 'itemresponses' will work (it could be student name). We are only interested in values (list). Try `list(student_list[0].values())`